SPECS/iptables/ip4save

```diff
@@ -2,12 +2,12 @@
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
-:OUTPUT DROP [0:0]
+:OUTPUT ACCEPT [0:0]
 # Allow local-only connections
 -A INPUT -i lo -j ACCEPT
 -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+-A INPUT -p icmp -j ACCEPT
```

Comment:
This will change the default configuration for all Mariner images, which may or may not be a significant change for some of our users. @AZaugg, @rlmenge, can you guarantee that this is a benign change? I'm not an expert in this topic, unfortunately, but it's best to be careful about such system-wide changes.

@AZaugg, why is this change necessary for all of Mariner?

Reply:
From my point of view it helps with debugging. We had a new server being provisioned and it included the iptables package, the system failed to build correctly and debugging was a lot more involved. ping inbound were being dropped, which lead us down several paths of "network" issues. with iptables, not the first place we looked.

In addition, certain icmp messages are important, for example, "icmp fragmentation required". I think it is used for MTU and MSS negotiation.

Reply:
Spoke with @chaitanyalala and this goes against our "secure by default" policy where customers (running on both bare metal or containers) should have the highest security possible by default and then can customize and accept ports / packets as needed.